### PR TITLE
refactor: Enables testpackage and nlreturn linters

### LINKED ISSUE #12: enable testpackage and nlreturn linters

### PR DESCRIPTION
**Pull request by soluchok**
This PR enables the following linters:
* testpackage
* nlreturn

Closes #12 

Signed-off-by: Andrii Soluk <isoluchok@gmail.com>

### CODE REVIEW

File: .golangci.yml [hunk 1]
```diff
@@ -74,9 +74,7 @@ linters-settings:
 linters:
   enable-all: true
   disable:
-    - testpackage # TODO enable
     - goerr113 # TODO enable
-    - nlreturn # TODO enable
```
Comment (llorllale) [inline .golangci.yml:79]: I would in all seriousness forget about `goerr113` (see https://github.com/Djarvur/go-err113/issues/10).... or at most, leave a TODO to followup on closure of that issue and _maybe_ enable it again.

Comment (soluchok) [reply .golangci.yml:79]: Removed TODO.

**Review by fqutishat — APPROVED** [review]

File: component/storage/couchdb/store.go [hunk 2]
```diff
@@ -191,7 +191,7 @@ func (p *Provider) Close() error {
 // StoreCouchDB represents a CouchDB-backed database.
 type StoreCouchDB struct {
 	log logger
-	db  *kivik.DB
+	DB  *kivik.DB
```
Comment (rolsonquadras) [inline component/storage/couchdb/store.go:194]: Just curious - which linter was complaining about having un-exported field ?

Comment (soluchok) [reply component/storage/couchdb/store.go:194]: It is about tests. There is no way to create an index. See https://github.com/hyperledger/aries-framework-go-ext/pull/13/files#diff-f23f647e13d912c2545a3a269868819dfc85656e02a8ce4072471e9c2e642bc9R219

Comment (llorllale) [reply component/storage/couchdb/store.go:194]: @soluchok the test can instantiate a separate `kivik.DB` instance and create the index with that.

**Review by rolsonquadras — APPROVED** [review]

**Review by llorllale — APPROVED** [review]

File: component/storage/couchdb/store.go (hunk 2 above)
Comment (soluchok) [reply component/storage/couchdb/store.go]: Fixed. One test was removed. Not the critical one but affects coverage a bit.

**Review by troyronda — APPROVED** [review]

**Pull request merged**